src/pages/informational/WpPlugin.tsx

```diff
@@ -0,0 +1,225 @@
+import character from "assets/laira/laira-waiving.png";
+import Copier from "components/Copier";
+import Image from "components/Image";
+
+export default function WpPlugin() {
+  return (
+    <div
```

Comment:
there's visible overflow in `WpPlugin` page on narrower screens `<550px`
<img width="559" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/cfa899fe-f846-4231-b2b0-35161a13401b">

Reply:
@ap-justin Addressed this item and the others in 1847ba37. Thanks!